core/src/test/java/hudson/FilePathTest.java

```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- * 
```

Comment:
All the formatting changes are not related. Please submit refactoring separately